### PR TITLE
Clean up for the new domain name changes.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* [15877](https://github.com/bcgov/entity/issues/15877)

*Description of changes:*
1. Clean up unused files;
2. Enable cloud build PR review feature;
3. Move firebase and cloud build configuration to UI CD flow in bcregisty-sre repo;
4. Update environment variables' value;

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: korc4xp.web.app

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bc-registry",
-  "version": "2.0.4",
+  "version": "2.1.0",
```
Comment (seeker25) [inline package.json:3]: major version upgrade and nothing has changed other than CI/CD :(

it's fine

**Review by seeker25 — APPROVED** [review]

**Review by cameron-eyds — APPROVED** [review]
LG!

**Review by doug-lovett — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (severinbeauvais) [reply package.json:3]: That's a minor version change and files were deleted.

**Pull request merged**